app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -35,6 +36,10 @@ def ems_inv_to_hashes
 
   private
 
+  def map_labels(model_name, labels)
```

Comment:
I think that might be confusing, given that the method called internally is `.map_labels`, and none of the *actual* mapping happens until post.